src/thermo/ThermoFactory.cpp

```diff
@@ -53,6 +54,9 @@ ThermoFactory::ThermoFactory()
     reg("ideal-surface", []() { return new SurfPhase(); });
     addAlias("ideal-surface", "Surface");
     addAlias("ideal-surface", "Surf");
+    reg("coverage-dependent-surface", []() { return new CoverageDependentSurfPhase(); });
+    addAlias("coverage-dependent-surface", "CoverageDependentSurface");
+    addAlias("coverage-dependent-surface", "CovDepSurf");
```

Comment:
These aliases don't need to be added -- the other naming styles are just for backwards compatibility with CTI/XML input files.